LaTeX/LaTeX.sublime-syntax

```diff
@@ -723,6 +706,14 @@ contexts:
           scope: punctuation.definition.verb.latex
           pop: true
 
+  xparse-args-bare-tokens:
+    - match: (\\){{cmdname}}{{endcs}}
+      scope: constant.other.token.latex
+      captures:
+        1: punctuation.definition.backslash.latex
+    # or a single character
+    - match: '\S'
+      scope: constant.character.latex constant.other.token.latex
```

Comment:
What is this unused context meant for?

Reply:
oh, I added this when I wanted to unify the embellishment and token parts. In principle, for a token argument, only one token is valid, whereas within braces you can specify multiple tokens for an embellishment. But as we want to balance braces anyway, it turns out that the braced token context handles multiple tokens in any case, so the two constructs actually can share the same context, so this turned out to be unnecessary.